package.json

```diff
@@ -2,22 +2,22 @@
   "name": "chubby",
   "version": "1.0.0",
   "description": "",
-  "main": "src/bot.js",
+  "main": "./dist/bot.js",
   "scripts": {
-    "lint": "./node_modules/.bin/eslint .",
     "start": "node .",
-    "start:dev": "NODE_ENV=development node ."
+    "start:dev": "NODE_ENV=development node .",
+    "lint": "npx eslint .",
+    "build": "npm run lint && npm run clean && npx tsc && npx tsc-alias",
+    "clean": "rimraf ./dist"
   },
   "keywords": [],
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "@discordjs/builders": "^0.13.0",
-    "@discordjs/rest": "^0.4.1",
```

Comment:
They're included in discordjs from v14 onward, it's part of the migration

Reply:
I see, this is actually a first for us. We've never encountered an instance here before where we had to use a library that was not explicitly installed and tracked as a top level dependency. We don't currently have a precedent for this, though I can see arguments for both sides

Not including it as a top level dependency removes "unnecessary" dependencies from the list, but on the other hand it can be confusing to developers as to *where* these packages come from. This should *probably* be discussed more tbh before we settle on one way or the other

I'm personally in favor of always explicitly marking dependencies in the dependency list, but again we don't have any precedent atm

Reply:
I think it depends on the library. For example if a library depended on a generic util library like lodash or something, I wouldn't want to depend on that transient dependency. In this case, those are basically subpackages of the library itself and it's recommended in the [migration guide](https://discordjs.guide/additional-info/changes-in-v14.html#various-packages-are-now-included-in-v14) not to include them separately so you don't get odd version conflicting behaviour. I think that makes sense in this case because from a domain perspective all these libraries share a domain

Reply:
Given the specific recommendation from DJS on this one, I'll say this is fine. Though we should establish some stronger guidelines for this at some point